helm/install/templates/manager.yaml

```
@@ -17,7 +17,7 @@ spec:
     spec:
       containers:
       - name: operator
-        image: "{{ .Values.image.image }}"
+        image: {{ .Values.controllerImages.cluster | required ".Values.controllerImages.cluster" | quote }}
```

When a required value isn't present, does it show just ".Values.controllerImages.cluster" as the error message?

---

Hmm, it does. `lint` prefixes the message, but not `install`.

```console
$ helm lint --set controllerImages.cluster= helm/install
engine.go:167: [INFO] Missing required value: .Values.controllerImages.cluster

$ helm install --set controllerImages.cluster= … helm/install
Error: INSTALLATION FAILED: execution error at (pgo/templates/manager.yaml:20:53): .Values.controllerImages.cluster
```

Is the following enough?

```suggestion
        image: {{ .Values.controllerImages.cluster | required ".Values.controllerImages.cluster is required" | quote }}
```

```console
$ helm lint --set controllerImages.cluster= helm/install
engine.go:167: [INFO] Missing required value: .Values.controllerImages.cluster is required

$ helm install --set controllerImages.cluster= … helm/install
Error: INSTALLATION FAILED: execution error at (pgo/templates/manager.yaml:20:53): .Values.controllerImages.cluster is required
```

---

Yeah, that looks good to me -- I imagine a lot more errors arise from bad installs